src/components/common/EffectiveDateTime.vue

```diff
@@ -462,4 +465,18 @@ export default class EffectiveDateTime extends Mixins(DateMixin) {
     color: $BCgovInputError !important;
   }
 }
+::v-deep {
+  .date-picker-form  .v-input__control > .v-input__slot:before {
+    border-image: none;
+  }
+  .theme--light.v-icon.v-icon.v-icon--disabled {
+    color: $app-blue !important;
+  }
+  .theme--light.v-input--is-disabled {
```

Comment:
I'm not sure if this rule applies to this component's v-inputs (if any?) or the ones in the date picker component, but if the latter then this probably shouldn't be here. (Also line 478.)

Reply:
Fixed. Now DatePicker has been changed.